backend/danswer/db/document_set.py

```diff
@@ -525,36 +525,52 @@ def fetch_document_sets_for_documents(
 ) -> Sequence[tuple[str, list[str]]]:
     """Gives back a list of (document_id, list[document_set_names]) tuples"""
     stmt = (
```

Comment:
Previously, this just wouldn't return a row at all for document that didn't have any document sets associated with them.

Reply:
outerjoin gang